packages/block-library/src/image/index.php

```diff
@@ -204,7 +204,7 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 	$button =
 		$img[0]
 		. '<button
-			class="lightbox-trigger"
+			class="wp-lightbox-trigger"
```

Comment:
At this point, it's probably safer to leave the class name unchanged as sites might have styles in place that depend on the old class name.

Reply:
Reverted 👍